File: Assignment_1/Assignment1_5.py
"""------------------------------------------------------------------------------
                            Assignment1_5
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
------------------------------------------------------------------------------
Problem statement :
            This program outputs number 10 to 1    
Learnings:
          ---> Use of "for loop" with range() in python
          ---> Decrementing counter
          ---> Printing output on same line end=" "
Special Variable used : end=" "          
Control statment used:
                   ---> for _______ in range(___,____,___) 

------------------------------------------------------------------------------"""
#----------------------------------------------------------------------
#  This function prints numbers in decreasing order
#----------------------------------------------------------------------
def PrintNoDecreasingOrder(num):
    print("----------------------------------------------")
    print("Decreasing order numbers from 10 to 1  : ")
    for no in range(num,0,-1):
        print(no,end=" ")  
    
    print("\n-------------------------------------------")
#-------------------------------------------------------------------
# This function accepts a number and 
# calls PrintNoDecreasingOrder(num) function
#----------------------------------------------------------------------
def main():
    number = 10
    PrintNoDecreasingOrder(number)
#------------------------------------------------------------------

#------------------------------------------------------------------  
# Entry point of program
if __name__=="__main__":
    main()
#----------------------------------------------------------------   

File: JavaPractice/Assignment_1/Assignment1_5.java
/*------------------------------------------------------------------------------
                            Assignment1_5
                    (Student name - Vaishali Jorwekar)
------------------------------------------------------------------------------
Problem statement :
            This program outputs number 10 to 1    
------------------------------------------------------------------------------*/
import java.util.Scanner;;
public class Assignment1_5 {

public void printDecreasingNumbers(int number){
    for(int cnt=number;cnt>0;cnt--){
        System.out.print(cnt+"  ");

    }
    System.out.println();
}    
public static void main(String args[]){
    Scanner inputObj=new Scanner(System.in);
    System.out.println("Enter the counter to display numbers:");
    int number = inputObj.nextInt();
    Assignment1_5 printNumObj =new Assignment1_5();
    printNumObj.printDecreasingNumbers(number);
    inputObj.close();

}
    
}
